### PR TITLE
Raise error in Page#find when it does not return a name of the page

### LINKED ISSUE #87: A URL should not be a valid Facebook page
This code

```ruby
Funky::Page.find('google.com/dfdf')
```

_correctly_ raises an error

```ruby
Funky::ContentNotFound: Error 400: {"error":{"message":"Unknown path components: \/dfdf","type":"OAuthException","code":2500,"fbtrace_id":"CRHmwoGL1pe"}}
```
 
while this code

```
Funky::Page.find('FoxMovies')
```

correctly works 

 ```
#<Funky::Page:0x007ff6a23362d8
 @data=
  {:name=>"20th Century Fox",
   :username=>"FoxMovies",
   :fan_count=>23584846,
   :featured_video=>{:description=>"Kingsman: The Golden Circle trailer coming tomorrow.", :updated_time=>"2017-04-23T13:07:26+0000", :id=>"1398396143549811"},
   :id=>"121150347941070"}>
```

However this code **should raise an error** (like the first example):

```
Funky::Page.find("https://www.facebook.com/pg/FoxMovies/videos/?ref=page_internal")
```

but instead returns:

 ```
=> #<Funky::Page:0x007ff6a1fc9f00
 @data=
  {:og_object=>{:id=>"1422854821081236", :type=>"website", :updated_time=>"2016-12-24T16:25:29+0000"},
   :share=>{:comment_count=>0, :share_count=>0},
   :id=>"https://www.facebook.com/pg/FoxMovies/videos/"}>
```

### PR DESCRIPTION
**Pull request by kangkyu**
closes #87

### CODE REVIEW

File: lib/funky/page.rb [hunk 1]
```diff
@@ -12,7 +12,11 @@ class Page < GraphRootNode
     # @return [Funky::Page] containing the data fetched by Facebook Graph API.
     def self.find(page_id)
       page = Funky::Connection::API.fetch("#{page_id}?fields=name,username,location,fan_count,featured_video")
-      new(page)
+      if page[:name].nil?
```
Comment (claudiofullscreen) [inline lib/funky/page.rb]: @kangkyu Out of curiosity, is there a `:type` field that we could use instead?

File: lib/funky/page.rb [hunk 2]
```diff
@@ -12,7 +12,11 @@ class Page < GraphRootNode
...
+        raise ContentNotFound, "Sorry, page did not return its name field"
```
Comment (claudiofullscreen) [inline lib/funky/page.rb]: @kangkyu 

```ruby
raise ContentNotFound, "Page not found with ID #{page_id}"
```

File: lib/funky/page.rb (hunk 1 above)
Comment (kangkyu) [inline lib/funky/page.rb]: "page" does not have `type` field. But there's documentation for "url" https://developers.facebook.com/docs/graph-api/reference/v2.8/url (and it has `type` field under `og_object`)

Comment (claudiofullscreen) [inline lib/funky/page.rb]: @kangkyu but *every* page has a `title`? Is that correct?

**Comment by claudiofullscreen** [comment]
@kangkyu If that's the case, feel free to bump the version, README, Changelog… 👍

File: lib/funky/page.rb (hunk 1 above)
Comment (kangkyu) [inline lib/funky/page.rb]: I think so. "The name of the Page" is one of their core fields
> We guarantee that those core APIs will be available

**Pull request merged**